pages.es/common/ansible-doc.md

```diff
@@ -0,0 +1,29 @@
+# ansible-doc
+
+> Muestra información sobre los módulos instalados en las bibliotecas de Ansible.
+> Muestra una concisa lista de complementos y sus breves descripciones.
+> Más información: <https://docs.ansible.com/ansible/latest/cli/ansible-doc.html>.
+
+- Lista de complementos disponibles acorde a su acción (módulos):
+
+`ansible-doc --list`
+
+- Lista de complementos disponibles dado un tipo específico:
+
+`ansible-doc --type {{tipo_complemento}} --list`
```

Comment:
```suggestion
- Lista de complementos disponibles dado un tipo específico:

`ansible-doc --type {{become|cache|callback|cliconf|connection|...}} --list`
```

Mandatory fix (error source: original page): [broken placeholder content](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#special-cases): expected 5 listed alternatives with ellipsis.

Reply:
So, we should update the original file too?

Reply:
Already done in #9739. :)